lib/google_crawler_web/templates/keyword/index.html.eex

```diff
@@ -18,9 +18,13 @@
       <div class="card card-body">
         <form>
           <div class="form-group row">
+            <div class="col-4">
+              <label for="title">Keyword Title</label>
+              <input class="form-control" name="keyword_filter[title]" placeholder='"apple", "rocket"' value=<%= @params["keyword_filter"]["title"] %> >
```

Comment:
Small thing about the placeholder, can we go without the `double quote` there @hanam1ni 

BEFORE
```
placeholder='"apple", "rocket"'
```

AFTER
```
placeholder="apple, rocket"
```

Reply:
Just because the user will confuse should them input the `"` in the input field or not :D

Reply:
Updated in 9ae01d7